beta/src/components/Icon/IconNewPage.tsx

```diff
@@ -5,14 +5,15 @@
 import * as React from 'react';
 
 export const IconNewPage = React.memo<JSX.IntrinsicElements['svg']>(
-  function IconNewPage({className}) {
+  function IconNewPage({className, ...rest}) {
```

Comment:
```suggestion
  function IconNewPage(props) {
```
Shall we spread the props completely? Instead of destructuring the className from props and then spreading the rest and then spreading the rest values in line 16 ?